index.js

```diff
@@ -39,6 +35,10 @@ http.createServer(function(req, res) {
   options.now = formatDate(new Date());
   options.fileName = sanitizeFileName(options.fileName);
```

Comment:
Oh, unfortunately I found a bug: if `options.fileName` is null, we'll try to run it through `sanitizeFileName` which will fail with `no method replace on null` or something. You can probably just fix it with `options.fileName = sanitizeFileName(options.fileName || '');`

(and at that point you can assume that `options.fileName` is a string, and remove the `!options.fileName` check on `L38`)